### ISSUE TITLE
config builder tool

### ISSUE THREAD

**Issue by davewhiteland** [mysociety/fixmystreet#554]
A wee tool (webpage, js?) that behaves a bit like a wizard, that builds a suggested conf/general.yml file

Many of those settings can be deduced from a simple question/answer, and the slightly fiddly ones (Mapit areas) are not so frightening once you understand what you need.

Features/benefits:
- some syntax checking
- warnings if values are unexpected
- extra help/links to the MAPIT stuff, which usually requires simple values but which is a bit confusing for newbies
- (obviously) for nervous self-hosters but (less obviously) I think I would use it too
- final yml won't have anything missing or misspelled (I had `MAPIT_WHITELIST` in one for a while, which doesn't work)

Basically taking the hints/facts from the general.yml-example file and pushing through them one by one, Q&A style, ending with a yml file (for copy and paste presumably) that is likely to be right.
